docs/contents/components/forms/dropzone/index.mdx

```diff
@@ -285,10 +343,24 @@ To handle only rejected files, use `onDropRejected`.
     </DropzoneIdle>
 
     <VStack gap="2xs">
-      <Text fontSize="xl">Drag images here or click to select files</Text>
-      <Text fontSize="sm">
-        Attach as many files as you like, each file should not exceed 3mb
-      </Text>
+      <For
```

Comment:
There is no need to use `For` here. I feel that the modification makes it less readable.

Reply:
Fixed.